### ISSUE TITLE
Bootstrap3ImageCMSPlugin in TextField (text mode) cannot be aligned using ckeditor's JustifyCenter, JustifyRight in DjangoCMS edit mode

### ISSUE THREAD

**Issue by macolo** [aldryn/aldryn-bootstrap3#88]
Bootstrap3ImageCMSPlugin creates HTML that contains div, but ckeditor's JustifyCenter or JustifyRight button wraps the selected node into a paragraph which creates invalid HTML (div inside a p).

Example of the code Bootstrap3ImageCMSPlugin produces:

```
<div class="js-filer-dropzone filer-dropzone" data-filer-url="/en/admin/cms/page/plugin/bootstrap3_image_cms_plugin/ajax_upload/697/"><img src="/media/filer_public/5b/d6/5bd6213c-abea-487a-9b4e-620bbc14457b/social.png" alt="" class="js-original-image img-responsive"><div class="filer-dropzone-info-message js-filer-dropzone-info-message hidden"><div class="filer-dropzone-icon"><span class="fa fa-cloud-upload"></span></div><div class="filer-dropzone-upload-welcome js-filer-dropzone-upload-welcome"><div class="text">Drop your file to change image:</div></div><div class="filer-dropzone-upload-info js-filer-dropzone-upload-info hidden"><div class="js-filer-dropzone-file-name filer-dropzone-file-name"></div><div class="filer-dropzone-progress js-filer-dropzone-progress"></div></div><div class="js-filer-dropzone-upload-success hidden">
                        Upload success!
                    </div></div><div class="filer-dropzone-error-message js-filer-dropzone-error-message hidden"><div class="icon"><span class="fa fa-cloud-upload"></span></div><div class="js-filer-dropzone-upload-accept filer-dropzone-text">
                        Error! Files of this type are not accepted.
                    </div></div></div>
```

**Comment by bharling** [comment]
+1 On our site this issue results in the widget being displayed over the top of the site constantly in edit mode. having looked at the markup I'm 90% sure this is the problem we're seeing.

**Comment by FinalAngel** [comment]
This will be fixed in the upcoming 1.2.0 release
